settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':app'
+rootProject.name = "AuthenticatorApp"
```

Comment:
this needs to be changed to GymBuddy. this will absolutely cause issues with the project if it is trying to reference two different names for it